src/libraries/System.Text.Json/tests/Common/ConstructorTests/ConstructorTests.Stream.cs

```diff
@@ -10,162 +10,114 @@ namespace System.Text.Json.Serialization.Tests
 {
     public abstract partial class ConstructorTests
     {
-        [Fact]
-        [SkipOnCoreClr("https://github.com/dotnet/runtime/issues/45464", ~RuntimeConfiguration.Release)]
-        public async Task ReadSimpleObjectAsync()
+        [Theory]
```

Comment:
Converted this test suite to use theories because it was causing me grief while debugging.